kingsleyreadlexicon.tsv

```diff
@@ -87912,8 +87912,8 @@ streets	𐑕𐑑𐑮𐑰𐑑𐑕	NN2	striːts	4941
 streetwalker	𐑕𐑑𐑮𐑰𐑑𐑢𐑷𐑒𐑼	NN1	striːtwɔːkəR	3
 streetwalkers	𐑕𐑑𐑮𐑰𐑑𐑢𐑷𐑒𐑼𐑟	NN2	striːtwɔːkəRz	4
 strength	𐑕𐑑𐑮𐑧𐑙𐑔	NN1	streŋθ	7077
-strengthen	𐑕𐑑𐑮𐑧𐑙𐑔𐑩𐑯	VVB	streŋθn	171
-strengthen	𐑕𐑑𐑮𐑧𐑙𐑔𐑩𐑯	VVI	streŋθn	1032
+strengthen	𐑕𐑑𐑮𐑧𐑙𐑔𐑩𐑯	VVB	streŋθƏn	171
+strengthen	𐑕𐑑𐑮𐑧𐑙𐑔𐑩𐑯	VVI	streŋθƏn	1032
```

Comment:
I suggest these should be the following for consistency, as I don't think there's any suggestion that RP speakers might say /streŋθɪn/:

```
strengthen	𐑕𐑑𐑮𐑧𐑙𐑔𐑩𐑯	VVB	streŋθən	171
strengthen	𐑕𐑑𐑮𐑧𐑙𐑔𐑩𐑯	VVI	streŋθən	1032
```